src/feedbacks.ts

```diff
@@ -0,0 +1,75 @@
+import { CompanionFeedbacks, CompanionFeedbackEvent, CompanionFeedbackResult } from "../../../instance_skel_types"
+import InstanceSkel = require("../../../instance_skel")
+import { MagewellConfig } from "./config"
+import { DeviceStatus } from "./magewell"
+import { MagewellState } from "./magewellstate"
+
+export enum FeedbackId {
+  Stream = 'stream',
+  Record = 'record'
+}
+
+export function GetFeedbacks(instance: InstanceSkel<MagewellConfig>, state: MagewellState): CompanionFeedbacks {
+  return {
+    [FeedbackId.Record]: {
+      label: 'Change color based on Record status',
+      description: 'If the device is recording change the color',
+      options: [
+        {
+          type: 'colorpicker',
+          label: 'Color Live',
+          id: 'color_live',
+          default: instance.rgb(222, 0, 0)
+        },
+        {
+          type: 'colorpicker',
+          label: 'Color Ready',
+          id: 'color_ready',
+          default: instance.rgb(209, 209, 0)
+        }
+      ],
+      callback: (evt: CompanionFeedbackEvent): CompanionFeedbackResult => {
+        if (!state.status) return {};
+        if ((state.status["cur-status"] & DeviceStatus.statusRecord) == DeviceStatus.statusRecord) {
+          return {
+            bgcolor: Number(evt.options.color_live)
+          }
+        } else {
+          return {
```

Comment:
You shouldnt do a `color_ready` like this. The same behaviour can be achieved by instead changing the 'main' background color of the button, and will let the user put multiple feedbacks on one button.

Reply:
I'm not entirely sure if this is what you mean, but I took a stab at it :)